CMakeLists.txt

```diff
@@ -34,6 +34,11 @@ set(CMAKE_MACOSX_RPATH 1)
 
 add_definitions("-DSQLITE_HAS_CODEC")
 
+# To compile with PostgreSQL support
+if (PG_SUPPORT)
+    add_definitions("-DPG_SUPPORT")
```

Comment:
That's the purpose of using a specific name for this macro, since it's affecting how the whole project is configured and compiling it should appear at top level (same as option `BUILD_TESTS`)